Tools/CISupport/ews-build/steps.py

```diff
@@ -533,8 +533,7 @@ def add_pr_details(self):
 
         self.setProperty('change_id', revision[:HASH_LENGTH_TO_DISPLAY], 'ConfigureBuild')
 
-        if title:
-            self.addURL('PR {}: {}'.format(pr_number, title), GitHub.pr_url(pr_number, repository_url))
+        self.addURL('PR {}{}'.format(pr_number, (': ' + title) if title else ''), GitHub.pr_url(pr_number, repository_url))
```

Comment:
For readability, let's avoid inline if else inside the function parameters. You can do something like:

```
title = ': ' + title if title else ''
self.addURL(...
```